include/sycl_ext_complex.hpp

```diff
@@ -450,13 +451,13 @@ class complex<_Tp, typename std::enable_if<is_genfloat<_Tp>::value>::type> {
     return __t;
   }
   _SYCL_EXT_CPLX_INLINE_VISIBILITY friend complex<value_type>
-  operator+(const complex<value_type> &__x, const value_type &__y) {
+  operator+(const complex<value_type> &__x, value_type __y) {
```

Comment:
Sorry stupid question, but I Don't find this function in `https://en.cppreference.com/w/cpp/numeric/complex/operator_arith`.  The only one I see, return `&complex<value_type>`

Reply:
So these `binary operators` were already in the implementation; they just have been moved within the class scope to be hidden friends. (`The befriended entity should be fully defined within the class synopsis`).

You should see it there: https://en.cppreference.com/w/cpp/numeric/complex/operator_arith3